app/src/components/tags/HorizontalScrollableTagViewer.vue

```diff
@@ -119,6 +122,7 @@ useResizeObserver(scrollContent, setSpinBtnVisibility);
                         v-for="content in taggedDocs"
                         :key="content._id"
                         :content="content"
+                        :show-publish-date="props.showPublishDate"
```

Comment:
`props` is not necessary here, as all props are available by their name in the Vue template.

```suggestion
                        :show-publish-date="showPublishDate"
```